release notes/upcoming.md

```diff
@@ -10,5 +10,4 @@ Description of feature.
 
 ## Bugs fixed!
 
-* HTTP: Use Request's GetBody in order to be able to get the body multiple times for a single
-  request as needed in 308 redirects of posts and if the server sends GOAWAY with no error (#1093)
+* Category: description of bug. (#PR)
```

Comment:
Given the way we handle release notes now, I'm wondering if we shouldn't just delete `upcoming.md`?

Reply:
Maybe ... moving it to `template.md`? With all the possible sections and maybe some additional guides ... like the dots ;)